Makefile

```diff
@@ -0,0 +1,17 @@
+SHELL := /usr/bin/env bash
+ARTIFACTORY_HOST := "artefacts.tax.service.gov.uk"
+tag = $$(cat .sbt-version)
```

Comment:
do we need an extra number in case we make changes that don't include an sbt change?

Reply:
I think we always want the tag to represent the SBT Version. If we update the Java version for example this just gets a new release in git tagged with latest and the sbt version it is just a new git commit. It is the same concept as docker-hmrc-jre